.semaphore/semaphore.yml

```diff
@@ -85,11 +45,11 @@ blocks:
       jobs:
         - name: "Build client"
           commands:
-            - "nix-build --no-out-link -E '(import ./nix/nixpkgs.nix {}).haskellPackages.callPackage ./client-haskell/client.nix {}'"
```

Comment:
could you also incorporate running `cabal-fmt` somewhere in the CI? (formatter for the cabal file)

Reply:
In the other repos it's part of `build.py` but we don't have a `build.py` here. Or any other lints for that matter. The CI just checks if it builds and the tests pass. There are currently no lints or style checks at all.